docs/advanced/input_files/input-main.md

```diff
@@ -739,7 +739,7 @@ calculations.
   ```
 
   Then the user has to correct the input file and restart the calculation.
-- **Default**: cg (pw) or genelpa (lcao)
+- **Default**: cg (plane-wave basis), or genelpa (localized atomic orbital basis)
```

Comment:
Please use indent for multi default, the write default for lcao is:
 - genelpa, (USE_ELPA has been set)
 - scalapack_gvx, (USE_ELPA has not been set)